src/java.base/share/classes/java/lang/invoke/CallSite.java

```diff
@@ -84,8 +84,8 @@ private static CallSite bootstrapDynamic(MethodHandles.Lookup caller, String nam
  * @author John Rose, JSR 292 EG
  * @since 1.7
  */
-abstract
-public class CallSite {
+public
+abstract class CallSite {
```

Comment:
I think it's better to move all modifiers to the single line.

Reply:
This is a survivorship bias. This example jumps out at you, because it happens to use missorted modifiers. I'm pretty sure this is not an aberration, but a style. If you want to change that style, you should create a respective JBS issue.

Reply:
I've grepped the code and can now see that a list of modifiers broken by newlines which cannot be explained by line-width concerns is indeed an irregularity. Not only in java.base but also in other modules. Although there aren't many of such cases, I would prefer them to be addressed in a separate cleanup, which you are welcome to pursue.